src/components/MediaPlayer.tsx

```diff
@@ -70,9 +71,18 @@ export function MediaPlayer(props: IProps) {
     audioRef.current.currentTime = 0;
     if (onEnded) onEnded();
   };
-
+  const handleError = (e: any) => {
+    console.log('MediaPlayer error', e);
```

Comment:
If you use the logError, you won't need this console log. The logging of the error handles this.